app/Models/Presentation.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class Presentation extends Model
+{
+    use HasFactory;
+
+    /**
+     * Returns the room that the presentation is in
```

Comment:
Empty comment line here

Reply:
Also, decide whether or not remove the "Returns the" phrase for these kinds of comments. Decide and then apply this consistently throughout the entire project (see `Booth.php`)